src/main/java/koksao/Main.java

```diff
@@ -7,7 +15,25 @@
 import java.util.Scanner;
 
 public class Main {
+
+
     public static void main(String[] args) {
+        Options options = new Options();
+        options.addOption("r", "rates", true, "path to a file");
+
+        CommandLineParser clp = new DefaultParser();
+        try {
+            CommandLine cl = clp.parse(options, args);
+            if (cl.hasOption("r")) {
+                String filePath = cl.getOptionValue("r");
+                args[0] = filePath;
+            } else {
+                System.out.println("Cannot find a file");
+            }
+        } catch (Exception e) {
+            e.printStackTrace();
```

Comment:
So you only printing stack trace in case of error? Program will probably won’t work without correct reading of arguments